components/credit-card/DebitCardWordProblemData.ts

```diff
@@ -0,0 +1,12 @@
+export const makeArray = ["make","take","give"];
```

Comment:
this shouldn't go in the components folder. this should go in a sub folder of the api folder